<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstPage</class>
 <widget class="QMainWindow" name="FirstPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FirstPage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="GB_FirstPage">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>521</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : white ;
border : 2px solid #3d3d3d ;
border-radius : 2px ;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_number_player">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none ;
font: 10pt &quot;B Sara&quot;;</string>
     </property>
     <property name="text">
      <string>بازی چند نفره باشه ؟</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_number_player">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid black ;
text-align : center ;
background-color : #d2dae2 ;</string>
     </property>
    </widget>
    <widget class="QLabel" name="welcome_lab">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none ;
font: 20pt &quot;Algerian&quot;;</string>
     </property>
     <property name="text">
      <string>         Welcome</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none ;
font: 10pt &quot;B Sara&quot;;</string>
     </property>
     <property name="text">
      <string>تغییر تم :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_theme">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid black ;
background-color : #d2dae2</string>
     </property>
    </widget>
    <widget class="QLabel" name="soldier1_lab">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>91</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none ;
image: url(:/soldier1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="soldier2_lab">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/soldier2.png);
border : none ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionnew_file"/>
    <addaction name="actionopen_file"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionundo"/>
    <addaction name="actionredo"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuview"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionnew_file">
   <property name="text">
    <string>new file</string>
   </property>
  </action>
  <action name="actionopen_file">
   <property name="text">
    <string>open file</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="text">
    <string>redo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
